saos-webapp/src/main/webapp/WEB-INF/tags/details/correctionJudgment.tag

```diff
@@ -0,0 +1,32 @@
+<%@ include file="/WEB-INF/view/common/taglibs.jsp" %>
+
+<%@ attribute name="correction" required="true" rtexprvalue="true" type="pl.edu.icm.saos.persistence.correction.model.JudgmentCorrection" %>
```

Comment:
I'm not sure if it's a good idea to have this particular correction in separate tag because the tags are not private and are visible everywhere. Do we want to see this tag everywhere even though we shouldn't use it separately?